experimental/report/renderer.go

```diff
@@ -901,6 +910,118 @@ func renderSidebar(bars, lineno, slashAt int, ss *styleSheet, multis []*multilin
 	return sidebar.String()
 }
 
+// suggestion renders a single suggestion window.
+func suggestion(snip snippet, startLine int, lineBarWidth int, ss *styleSheet, out *strings.Builder) {
+	out.WriteString(ss.nAccent)
+	padBy(out, lineBarWidth)
+	out.WriteString("help: ")
+	out.WriteString(snip.message)
+
+	// Add a blank line after the file. This gives the diagnostic window some
+	// visual breathing room.
+	out.WriteByte('\n')
+	padBy(out, lineBarWidth)
+	out.WriteString(" | ")
+
+	// When the suggestion spans multiple lines, we don't bother doing a by-the-rune
+	// diff, because the result can be hard for users to understand how to apply
+	// to their code.
```

Comment:
Per other comment above: I think it would be more clear to _always_ use the multi-line rendering strategy.

Reply:
As mentioned above, I've made it so that the single-line strategy is only used for pure additions.